tests/test_ops.py

```diff
@@ -21,155 +21,195 @@
 
 import pennylane
 from pennylane import numpy as np
-from pennylane.ops import cv
+from pennylane.ops import cv, qubit
 
 
 s_vals = np.linspace(-3, 3, 13)
 phis = np.linspace(-2 * np.pi, 2 * np.pi, 11)
 mags = np.linspace(0.0, 1.0, 7)
 
-
-@pytest.mark.parametrize("phi", phis)
-def test_rotation_heisenberg(phi):
-    """ops: Tests the Heisenberg representation of the Rotation gate."""
-    matrix = cv.Rotation._heisenberg_rep([phi])
-    true_matrix = np.array(
-        [[1, 0, 0], [0, np.cos(phi), -np.sin(phi)], [0, np.sin(phi), np.cos(phi)]]
-    )
-    assert np.allclose(matrix, true_matrix)
-
-
-@pytest.mark.parametrize("phi", phis)
-@pytest.mark.parametrize("mag", mags)
-def test_squeezing_heisenberg(phi, mag):
-    """ops: Tests the Heisenberg representation of the Squeezing gate."""
-    r = mag
-    matrix = cv.Squeezing._heisenberg_rep([r, phi])
-    true_matrix = np.array(
-        [
-            [1, 0, 0],
-            [0, np.cosh(r) - np.cos(phi) * np.sinh(r), -np.sin(phi) * np.sinh(r)],
-            [0, -np.sin(phi) * np.sinh(r), np.cosh(r) + np.cos(phi) * np.sinh(r)],
-        ]
-    )
-    assert np.allclose(matrix, true_matrix)
-
-
-@pytest.mark.parametrize("phi", phis)
-@pytest.mark.parametrize("mag", mags)
-def test_displacement_heisenberg(phi, mag):
-    """ops: Tests the Heisenberg representation of the Displacement gate."""
-    r = mag
-    hbar = 2
-    matrix = cv.Displacement._heisenberg_rep([r, phi])
-    true_matrix = np.array(
-        [
-            [1, 0, 0],
-            [np.sqrt(2 * hbar) * r * np.cos(phi), 1, 0],
-            [np.sqrt(2 * hbar) * r * np.sin(phi), 0, 1],
-        ]
-    )
-    assert np.allclose(matrix, true_matrix)
-
-
-@pytest.mark.parametrize("phi", phis)
-@pytest.mark.parametrize("theta", phis)
-def test_beamsplitter_heisenberg(phi, theta):
-    """ops: Tests the Heisenberg representation of the Beamsplitter gate."""
-    matrix = cv.Beamsplitter._heisenberg_rep([theta, phi])
-    true_matrix = np.array(
-        [
-            [1, 0, 0, 0, 0],
+class TestQubit:
```

Comment:
Maybe it's better to make a folder, `ops/test_ops_qubit.py`? The test directory is starting to get very hard to find the right tests :laughing: